<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>parametre</class>
 <widget class="QDialog" name="parametre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>30</y>
     <width>157</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Couleur du fond d'écran :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_noir">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>60</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(64, 64, 64);
 border-style: solid;
 border-width:1px;
 border-radius:11.5px;
 border-color: white;
 max-width:23px;
 max-height:23px;
 min-width:23px;
 min-height:23px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_blanc">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
 border-style: solid;
 border-width:1px;
 border-radius:11.5px;
 border-color: rgb(64,64,64);
 max-width:23px;
 max-height:23px;
 min-width:23px;
 min-height:23px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>95</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Délai avant la photo :</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="TB_timer">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>125</y>
     <width>35</width>
     <height>32</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_valider">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 7px;
background-color: rgb(225, 225, 225);</string>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
